packages/react-form-state/src/test/FormState.test.tsx

```diff
@@ -124,6 +124,36 @@ describe('<FormState />', () => {
     });
   });
 
+  describe('reset()', () => {
+    it('resets all fields to their initial values', () => {
+      const renderPropSpy = jest.fn(() => null);
+      const product = faker.commerce.productName();
+
+      const form = mount(
+        <FormState initialValues={{product}}>{renderPropSpy}</FormState>,
+      );
+
+      const formDetails = lastCallArgs(renderPropSpy);
+      formDetails.fields.product.onChange(faker.commerce.productName());
+      form.update();
+
+      const {reset} = lastCallArgs(renderPropSpy);
+      reset();
+
+      form.update();
+
+      const {fields} = lastCallArgs(renderPropSpy);
+      reset();
+      expect(fields).toMatchObject({
+        product: {
+          value: product,
+          initialValue: product,
+          dirty: false,
+        },
+      });
+    });
+  });
```

Comment:
Are any of the above tests actually related to the changes you are making?

Reply:
they're just tests that were missing

Reply:
Actually I think they were partially some kind of rebase issue